mmpose/apis/inference.py

```diff
@@ -565,7 +565,11 @@ def inference_bottom_up_pose_model(model,
             })
 
         # pose nms
-        keep = oks_nms(pose_results, pose_nms_thr, sigmas=None)
```

Comment:
Please rebase to the latest webcam_api branch, which already handles the sigma issue